android/build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-    classpath 'com.android.tools.build:gradle:2.1.0'
+    classpath 'com.android.tools.build:gradle:7.2.2'
```

Comment:
Didn't work when this was 7.3.3 like the URL in gradle-wrapper.properties. I am not sure why, I am not an Android dev. Happy to update either if someone knows the right thing to do.